src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Resolvers/StaticCompositeResolver.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using MessagePack.Formatters;
+
+namespace MessagePack.Resolvers
+{
+    public class StaticCompositeResolver : IFormatterResolver
```

Comment:
Add an xml doc comment with a `<summary>` that explains the use for this sounds good. It should contrast this with `CompositeResolver` and explain how *this* class is for MPC consumers and applies app-wide.